source/codegen/metadata/nidcpower/CHANGES.md

```diff
@@ -1,7 +1,35 @@
-# Changes in metadata from the nimi-python metadata
+# Changes in metadata from the base metadata
+
+## config.py
+
+The following metadata was added:
+- 'c_function_prefix': 'niDCPower_'
+- 'service_class_prefix': 'NiDCPower'
+- 'java_package': 'com.ni.grpc.dcpower'
+- 'csharp_namespace': 'NationalInstruments.Grpc.DCPower'
+
+The following function name was capitalised:
+- 'close_function': 'Close'
+
+## functions.py
+
+The following functions were tagged with 'init_method': True, to ensure their generated service handler registers the new session with the session_repository:
+- InitializeWithIndependentChannels
+- InitializeWithChannels
+- InitExtCal
+
+The following functions were tagged with their corresponding c_names:
+- reset
+- self_test
+- revision_query
+- ConfigureOVP
+- error_message
+- error_query
 
```

Comment:
Note: The Changes to MeasureMultiple API (which was using source-code mechanism) will be updated into Changes.md file in the [next task ](https://ni.visualstudio.com/DevCentral/_workitems/edit/1426043)of providing implementation for that API. Because in that next task, we will be marking it with CustomCode tag.